src/govuk/helpers/_links.scss

```
@@ -382,10 +396,10 @@
   }
 }
 
-/// Print friendly link mixin
+/// Include link destination when printing the page
 ///
-/// When printing, append the the destination URL to the link text, as long
-/// as the URL starts with either `/`, `http://` or `https://`.
+/// If the user prints the page, add the destination URL after the link text, as
+/// long as the URL starts with either `/`, `http://` or `https://`.
```

```suggestion
/// long as the URL starts with `/`, `http://` or `https://`.
```
I think we only use 'either' with binary 'either/or' choices.